stag-library-compiler/src/main/java/com/vimeo/stag/processor/utils/KnownTypeAdapterFactoriesUtils.java

```diff
@@ -38,11 +41,15 @@
 import javax.lang.model.element.TypeElement;
 import javax.lang.model.util.Elements;
 
-public class KnownTypeAdapterFactoriesUtils {
+public final class KnownTypeAdapterFactoriesUtils {
 
     private static final String KNOWN_FACTORIES_RESOURCE = "StagTypeAdapterFactory.list";
 
-    public static Set<String> loadKnownTypes(ProcessingEnvironment processingEnv) throws IOException {
+    private KnownTypeAdapterFactoriesUtils() {
+    }
```

Comment:
Why throw exceptions when instantiating other final classes with static methods, but not with this class?